apps/arweave/src/ar_nonce_limiter.erl

```diff
@@ -303,7 +303,7 @@ request_validation(H, #nonce_limiter_info{ output = Output,
 			spawn(fun() -> ar_events:send(nonce_limiter, {invalid, H, 2}) end);
 		{RemainingStepsToValidate, NumAlreadyComputed}
 		  		when StartStepNumber + NumAlreadyComputed < StepNumber ->
-			case ar_config:use_remote_vdf_server() of
+			case ar_config:use_remote_vdf_server() and not ar_config:run_hybrid_vdf_mode() of
```

Comment:
Our intent is to basically have 3 different modes that can be mixed and matched, right?
1. Compute VDF locally (e.g. `compute_vdf`)
2. Share VDF with clients (e.g. `vdf_server`)
3. Get VDF from servers (e.g. `vdf_client`)

I'm having a little trouble projecting those 3 modes onto our legacy flags/config. E.g. where does `run_hybrid_vdf_mode()` fit in? Does it imply `vdf_server andalso vdf_client`?

Is it worth reworking our flags to match up with the new set of independent configurations? Or is it really just a "me problem" and I should figure out how to map the old config onto the new "hybrid mode"?

Reply:
What do you think about renaming it to "compute_own_vdf", on by default when no VDF servers are specified, off by default otherwise?